python/foglamp/plugins/south/sinusoid/sinusoid.py

```diff
@@ -25,14 +25,22 @@
 
 _DEFAULT_CONFIG = {
     'plugin': {
-        'description': 'Sinusoid async plugin',
+        'description': 'Sinusoid Plugin',
         'type': 'string',
-        'default': 'sinusoid'
+        'default': 'sinusoid',
+        'readonly': 'true'
+    },
+    'assetName': {
```

Comment:
make sure to have it in plugin_reconfigure as`if 'dataPointsPerSec' in diff or 'assetName' in diff:` 

```
# Plugin should re-initialize and restart if key configuration is changed
if 'dataPointsPerSec' in diff:
```